types/aws-param-store/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for aws-param-store 2.1
+// Project: https://github.com/vandium-io/aws-param-store#readme
+// Definitions by: Jason Gray <https://github.com/jasonthomasgray>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
```

Comment:
Is this needed?

Reply:
Without this I was getting a "cannot find name 'Buffer'" error.
I assume that the aws-sdk is using some node types.

Is there a more correct way for me to solve this issue?

Reply:
Hmm I think it's ok since aws-sdk needs a dependency on node to begin with.